Changelog.md

```diff
@@ -4,6 +4,9 @@
 
 - Improve test coverage of `Hermes::Logger::ParamsFilter` - cover regular expressions
 
+## 0.7.3
+- Handle database reconnection in synchronous flow
```

Comment:
this should be under `master` branch section, there is one more commit as you can see based on the message above.